index.html

```diff
@@ -0,0 +1,202 @@
+        </li>
+        <li>
+          <img src="img/peace icon.svg">
+        </li>
+      </ul>
+    </section><br><br><br><br><br><br><br><br>
+  </div>
+
+
+  <div>
+    <section class="together">
+      <img src="img/Snapshoot Portfolio.svg">
+      <div class="textp">
+        <h2>Tonic</h2>
+        <p class="inline-paragraph" id="canopy">CANOPY</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="devend">Back End Dev</p>
+        <img src="img/dot.svg">
+        <p class="inline-paragraph" id="unique">2023</p>
+      </div>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section class="multipost">
+      <img src="img/multipost.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+
+
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/tonic2.svg" class="postsvg">
+      <h2>Tonic</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
+
+
+    <section>
+      <img src="img/multi2.svg" class="postsvg">
+      <h2>Multi-Post<br>
+        Stories</h2><br>
+      <p class="inline-paragraph" id="canopy">CANOPY</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">Back End Dev</p>
+      <img src="img/dot.svg">
+      <p class="inline-paragraph">2023</p>
+      <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+      <ul>
+        <li class="my-button">HTML</li>
+        <li class="my-button">CSS</li>
+        <li class="my-button js">Javascript</li><br><br>
+      </ul>
+      <button type="button" class="see-project">See Project</button>
+    </section>
```

Comment:
- [ ] According to your previouse reviewer kindly Please make sure you added the hover and active properties to your button. Kindly check the Figma design,The button (e.g., "Get my resume,See project") is a link `a` or `button` tag, and follow the button styling: Active, Hover and Pressed (see design.
- [ ] ![Screenshot 2023-06-04 000218](https://github.com/Edgahkipkemoi/mobile/assets/92458236/e909995b-4946-403c-8285-ab721d671e93)
- [ ] see screenrecorf  

https://github.com/Edgahkipkemoi/mobile/assets/92458236/87f3ff1c-087a-4849-b717-a889596b1628